src/main/java/org/ice4j/util/AsyncQueueHandler.java

```diff
@@ -0,0 +1,285 @@
+/*
+ * Copyright @ 2018 Jitsi.org
```

Comment:
ok @mstyura last change: can you make this say `8x8, Inc.` instead of `Jitsi.org` here?

Reply:
Done :)